fuse/build.sh

```diff
@@ -13,4 +13,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Fuse source code should be a different change.

Reply:
Sure. Removed and will be uploaded in a separated patch. See above.